compat10/pglogical_compat.h

```diff
@@ -111,4 +111,17 @@ replorigin_drop_by_name(char *name, bool missing_ok, bool nowait)
 		replorigin_drop(originid, nowait);
 }
 
+#define PGLreplorigin_session_setup(node) replorigin_session_setup(node)
+
+#define PGL_PG_TRY(...) PG_TRY()
+#define PGL_PG_CATCH(...) PG_CATCH()
+#define PGL_PG_FINALLY(...) PG_FINALLY()
+#define PGL_PG_END_TRY(...) PG_END_TRY()
```

Comment:
These don't appear to be used/needed anywhere.

Reply:
Indeed, that seems to be some kind of copy pasto across all of the compat files

Reply:
Ops. I removed these unnecessary bits.